### PR TITLE
Review changes

### PR DESCRIPTION
**Pull request by krzysztofjanowski**

### CODE REVIEW

File: ch3_pull-request/src/app.js [hunk 1]
```diff
@@ -1,5 +1,14 @@
 function toRomanNumerals(num) {
-  return num;
+    var lookup = { M:1000,CM:900,D:500,CD:400,C:100,XC:90,L:50,XL:40,X:10,IX:9,V:5,IV:4,I:1 };
```
Comment (krzysztofjanowski) [inline ch3_pull-request/src/app.js:2]:
```suggestion
    var lookup = { M:1000,CM:900,D:500,CD:400,C:100,XC:90,L:50,XL:40,X:10,IX:9,V:5,IV:4,I:1 };
```

File: ch3_pull-request/src/app.js [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+    var roman = '';
+    var i;
+    for ( i in lookup ) {
```
Comment (krzysztofjanowski) [inline ch3_pull-request/src/app.js:5]:
```suggestion
    for ( i in lookup ) {
```

**Comment by wulfland** [comment]
You are supposed to do this in a fork of the repo. But you are not the first one 😅 Next edition of the book, I'm gonna will make this more explizit.

**Comment by wulfland** [comment]
1.	Open the following repository and **create a fork by clicking `Fork` in the top right corner** of the repository: https://github.com/wulfland/AccelerateDevOps
In the fork, navigate to Chapter 3 | Create a pull request (ch3_pull-request/Create-PullRequest.md). The file also contains the instructions so that you don’t have to switch between your browser and the book all the time.

**Pull request closed without merging**